src/Resources/contao/controllers/AutoBackupDB.php

```diff
@@ -21,23 +21,8 @@
 // Copyright (c) 2007-2021 by Softleister
 //-------------------------------------------------------------------
 
-class AutoBackupDb extends \Frontend
+class AutoBackupDb
 {
-    //-------------------------
-    //  Constructor
-    //-------------------------
-    public function __construct( )
-    {
-        parent::__construct( );                         // Construktor Backend ausführen
-
-        if( !defined( 'BE_USER_LOGGED_IN' ) ) {
-            define( 'BE_USER_LOGGED_IN', false );
-        }
-        if( !defined( 'FE_USER_LOGGED_IN' ) ) {
-            define( 'FE_USER_LOGGED_IN', false );
-        }
-    }
```

Comment:
_Note:_ these constants are already set by `ContaoFramework::initialize` and thus they do not have to be set here. Also you were extending from `Contao\Frontend` - but did not actually use any features of those parent classes so I removed that as well.

Reply:
Hallo Fritz,
leider ist die Fehlermeldung noch unverändert:
```
[2022-02-22T10:29:29.285888+01:00] request.CRITICAL: Uncaught PHP Exception LogicException: ""Softleister\BackupDbBundle\Controller\BackupDbController" has no container set, did you forget to define it as a service subscriber?" at / ... /vendor/symfony/framework-bundle/Controller/ControllerResolver.php line 36 {"exception":"[object] (LogicException(code: 0): \"Softleister\\BackupDbBundle\\Controller\\BackupDbController\" has no container set, did you forget to define it as a service subscriber? at / ... /vendor/symfony/framework-bundle/Controller/ControllerResolver.php:36)"} []
```